test/templates/table/component.njk

```diff
@@ -41,36 +41,36 @@ table:
 <div class="fd-table--fixed">
 {% endif %}
 <table  class="fd-table{{ modifier.block | modifier('table') }}{{ state | state }}{{ classes | classes }}"{{ aria | aria }}>
-    <thead>
-        <tr>
+    <thead class="fd-table__header">
+        <tr class="fd-table__header-row fd-table__row">
```

Comment:
Can we simplify these class names at all? So it only requires 1? Like since it's the child of `fd-table__header`, can we just have `fd-table__row` class and handle that in the scss?